Options/WeakAurasOptions.lua

```diff
@@ -7093,7 +7093,7 @@ function WeakAuras.CreateFrame()
     self.oldOnTextChanged = texteditorbox.editBox:GetScript("OnTextChanged");
     texteditorbox.editBox:SetScript("OnTextChanged", function(...)
       local str = texteditorbox.editBox:GetText();
-      if not(str) then
+      if not(str) or texteditorbox.combinedText == true then
```

Comment:
This is also new, and prevents any errors from showing if a combined text is edited.